selectrum.el

```diff
@@ -1882,13 +1882,24 @@ at the front.
 This is an `:around' advice for `minibuffer-message'. FUNC and
 ARGS are standard as in all `:around' advice."
   (if (bound-and-true-p selectrum-active-p)
-      (cl-letf* ((orig-put-text-property (symbol-function #'put-text-property))
-                 ((symbol-function #'put-text-property)
-                  (lambda (beg end key val &rest args)
-                    (apply orig-put-text-property
-                           beg end key (if (eq key 'cursor) 1 val)
-                           args))))
-        (apply func args))
+      (run-at-time
```

Comment:
Nice trick. I used to use `run-with-idle-timer`, but this is more semantically appropriate.

Reply:
It's a simple trick to solve such timing issues. Has saved me many times!